features/security/src/main/feature/feature.xml

```diff
@@ -114,6 +115,7 @@ org.ops4j.pax.web.default.realmname=DDF
         <bundle>mvn:javax.el/javax.el-api/3.0.0</bundle>
         <feature>pax-jetty</feature>
         <feature>session-invalidator</feature>
+        <feature prerequisite="true">wrap</feature>
```

Comment:
❓ What's with all these "wrap" additions?

Reply:
I'm assuming they are there to make sure the wrap feature is included before attempting to wrap a bundle.

That way if we want to start the bundle in isolation, it would work because we aren't depending on the wrap feature being pulled in transitively.

Reply:
Yeah these were existing errors but weren't uncovered until I was stepping through these tests one by one. I'm not sure why they didn't show up previously. This should have always been a failure.